{{ cookiecutter.project_name }}/{{ cookiecutter.project_name }}/base/templates/{{ cookiecutter.project_name }}/home.jinja

```diff
@@ -35,6 +35,9 @@
       <h1>mozilla webdev</h1>
     </header>
     It works! Now build something.
+    {% raw %}
+      {% csrf_token %}
+    {% endraw %}
```

Comment:
I totally don't understand this. Why does that work? I've only used `{% raw %}` when my django templates needs to spit out something like `{{ myvariable }}` for the sake of some angularjs code.